Resources/Prototypes/_Impstation/Entities/Clothing/Neck/cloaks.yml

```diff
@@ -80,3 +80,14 @@
     sprite: _Impstation/Clothing/Neck/Cloaks/hosgreatcoat.rsi
   - type: StealTarget
     stealGroup: HeadCloak
+
+- type: entity
+  parent: [ClothingNeckBase, BaseCommandContraband]
+  id: ClothingNeckCloakCmosgreatcloak
+  name: chief medical officer's great cloak
```

Comment:
pedantic, but the other greatcoats are one word: i suppose this is another type of Thing, but greatcloak might be more in-line

Reply:
it's not a great (adjective) cloak, it's a greatcloak. agreeing w dark that it would feel better if it were one word

Reply:
Already been changed to one word I agree as well beacuse it matches up with Greycloak as well